lib/member_page.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class String
```

Comment:
If you upgrade to the latest Scraped, it'll provide this for you: 
  https://github.com/everypolitician/scraped/commit/1628b52207fa9c29aeff1d491bdc631815c25a8f